### PR TITLE
reapplies fix for flyctrl not being a command

### PR DESCRIPTION
**Pull request by Steddy1Love**
## Did this break anything?
- [x] No
- [ ] Yes
    [ ] This broke some stuff
    [ ] This broke everything

## Type of change

- [ ] New feature (non-breaking change which adds functionality)
- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] Refactor(DRY-ing up/ reorganizing code, etc.)
- [x] Super small fix (Corrected a typo, removed a comment, etc.)
- [ ] Skip all the other stuff and briefly explain the fix.

## Requested feedback: 
- [ ] Yes
    I'd like feedback on (specify line #)... 
- [x] No

## Checklist:
- [x] If this code needs to be tested, all tests are passing
- [ ] The code runs locally
- [ ] There are comments where clarification/ organization is needed
- [ ] The code is DRY. If it's not, I tried my best
- [ ] I reviewed my code before pushing

## Testing Changes
- [x] No Tests have been changed
- [ ] Some Tests have been changed
- [ ] All of the Tests have been changed(Please describe what in the world happened)

## What's next?
Connect frontend

### CODE REVIEW

**Comment by Steddy1Love** [comment]
Just posting here as well, circleci is linked in GitHub to the wrong project in circleci. Working on fixing to correct to the updated listed project in circleci.

**Comment by Steddy1Love** [comment]
proof:
image

File: .circleci/config.yml
Comment (NicoShanstrom) [inline .circleci/config.yml:1]: ah man yeah that command for fly.

**Review by NicoShanstrom — APPROVED** [review]
good thats the fix!

**Comment by NicoShanstrom** [comment]
circleCI failing though

**Comment by NicoShanstrom** [comment]
getting this circleCI failure:
```
1) AdviceGenerator#check_to_call_api returns a recommendation when budget does not meet the 50/30/20 rule
     Failure/Error:
       response = connection.post(url) do |request|
         request.headers['Content-Type'] = 'application/json'
         request.body = params.to_json
       end
     
     Addressable::URI::InvalidURIError:
       Absolute URI missing hierarchical segment: 'http://:80'
     # ./app/services/fastapi_service.rb:19:in `call_api'
     # ./app/services/fastapi_service.rb:12:in `generate_advice'
     # ./app/poros/advice_generator.rb:15:in `check_to_call_api'
     # ./spec/requests/api/v1/advices_spec.rb:30:in `block (4 levels) in <top (required)>'
     # ./spec/requests/api/v1/advices_spec.rb:29:in `block (3 levels) in <top (required)>'

Finished in 0.10883 seconds (files took 5.61 seconds to load)
8 examples, 1 failure
```

**Pull request merged**